content/en/docs/instrumentation/java/manual.md

```diff
@@ -35,6 +35,87 @@ the most important piece of telemetry source-identifying info.
 
 For example:
 
+<details>
+  <summary>Maven top level dependencies. Click me.</summary>
```

Comment:
We always post both maven and gradle examples, e.g. see https://opentelemetry.io/docs/instrumentation/java/#maven

Can you add a gradle sample too, and make it look similar to the other pages? While I think it makes perfect sense to keep imports hidden in a `<details>` tag, we should be consistent with the build system samples.

Reply:
@mateuszrzeszutek I am wondering if that section you linked (https://opentelemetry.io/docs/instrumentation/java/#maven) should be moved to the /manual page? It is not needed for automatic instrumentation right? So I find it kind of disconnected, I was not even aware that it's there until recently...

Reply:
Hmm, yeah, it probably should -- although it might be needed in automatic instrumentation if you want to supplement it with your own API usage.
Maybe we should have a small, separate section about using the API together with the agent? E.g. that you need to use `GlobalOpenTelemetry` and so on. 
Definitely out of scope for this PR, though.

Reply:
yes, out of scope, but worth to look into later.

Reply:
@odupuy can you make the maven & gradle examples look similar to other pages (e.g. [this one](https://opentelemetry.io/docs/instrumentation/java/#maven))?